rotkehlchen/tests/api/test_history_base_entry.py

```diff
@@ -109,24 +112,39 @@ def _add_entries(server) -> list[HistoryBaseEntry]:
         counterparty='0x0EbD2E2130b73107d0C45fF2E16c93E7e2e10e3a',
     )]
 
+    entries_added_using_api = 0
     for entry in entries:
-        json_data = entry_to_input_dict(entry, include_identifier=False)
-        response = requests.put(
-            api_url_for(server, 'historybaseentryresource'),
-            json=json_data,
-        )
-        result = assert_proper_response_with_result(response)
-        assert 'identifier' in result
-        entry.identifier = result['identifier']
+        if entry.extra_data is not None:
+            # If any entry has extra data add it directly to the database instead of using
```

Comment:
why? And remember. If I have to ask that then that means the comment has to explain it too